stable/nginx-ldapauth-proxy/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: nginx proxy with ldapauth
 name: nginx-ldapauth-proxy
 icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png
-version: 0.1.2
+version: 0.1.3
 appVersion: 1.13.5
 sources:
```

Comment:
Please add `home: https://github.com/nginxinc/nginx-ldap-auth` filed here, that could allow your PR test to succeed.